### ISSUE TITLE
test framework

### ISSUE THREAD

**Issue by schriftgestalt** [googlefonts/glyphsLib#1003]
Some test use unittest.TestCase, others pytest. The later has the option to use pytest.mark.parametrize. Some unittest cases could use a parameter (e.g. to run it with a glyphs2 and glyphs3 file). Should I change the test to pytest or find another way?

**Comment by anthrotype** [comment]
> Should I change the test to pytest or find another way?

yes, that'd be preferable. New tests should use pytest by default. Old tests can be translated as needed

**Comment by schriftgestalt** [comment]
Thanks
